easy-ui-tokens/src/breakpoints/base.json

```diff
@@ -1,13 +1,10 @@
 {
-  "breakpoints": {
+  "breakpoint": {
     "xs": { "value": "0px" },
     "sm": { "value": "576px" },
     "md": { "value": "768px" },
     "lg": { "value": "992px" },
     "xl": { "value": "1200px" },
-    "xxl": { "value": "1400px" },
-    "mobile": { "value": "{breakpoints.xs}" },
-    "tablet": { "value": "{breakpoints.md}" },
-    "desktop": { "value": "{breakpoints.xl}" }
+    "xxl": { "value": "1400px" }
```

Comment:
removed the mobile, tablet, desktop aliases for now. will reassess if/how to incorporate once we revisit these with design